src/components/Nav/Nav.js

```diff
@@ -1,10 +1,50 @@
 import React from 'react';
-
+import './Nav.scss';
 class Nav extends React.Component {
   render() {
     return (
       <div className="Nav">
```

Comment:
- 대문자로 첫 글자를 시작하는 건, 컴포넌트 이름을 네이밍 할때만 입니다~ 클래스네임을 짓는 건 보통의 카멜케이스로 소문자로 시작해주세요!
```suggestion
      <div className="nav">
```
- 근데 사실, 이 div 태그는 불필요한 것 같습니다. 추가적인 스타일을 부여하기 위해서 사용하고 계신 것도 아니고, 가장 바깥에서 한가지 태그로 감싸기 위해서 사용하는 것도 아닌 것 같아서요! (이미 nav 태그로 가장 바깥에서 감싸주고 계십니다!)

Reply:
네!! 소문자로 수정했습니다!